b2share/modules/records/triggers.py

```diff
@@ -48,3 +50,14 @@ def set_record_schema(record, **kwargs):
         raise InvalidRecordError('Community ID is not a valid UUID.') from e
     schema = CommunitySchema.get_community_schema(community_id)
     record['$schema'] = community_schema_json_schema_link(schema)
+
+
+def index_record(record, **kwargs):
+    current_search_client.index(
+        index='records',
+        doc_type='record',
+        id=record.id,
+        body=record,
```

Comment:
Later we might also need to index *internal* metadata fields like `created` and `updated`.